content/before-you-open-a-support-ticket.md

```diff
@@ -11,6 +11,9 @@ keywords:
  - self-help
  - tech-support
  - troubleshooting
+ - repair guide
+ - internal view
+ - disassembly
```

Comment:
Should the keywords be in this article as well? It seems to me that it'd be better if they were only in the guides article